Corona-Energy-Grid-Frontend/src/app/REST/authentication.service.ts

```diff
@@ -42,7 +45,8 @@ export class AuthenticationService {
 
   //* Get authorization token */
   public getAuthorizationToken(): string {
-    return localStorage.getItem(AppConfig.LocalStorageKeys.TOKEN);
+    //return localStorage.getItem(AppConfig.LocalStorageKeys.TOKEN);
```

Comment:
comment?

Reply:
is het nodig om te veranderen van localstorage naar cookies als beiden op dezelfde manier functioneren?